src/components/Typography/BodyX/BodyX.test.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2023 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { render, screen } from '@testing-library/react'
+
+import { customCopyable, customEllipsis, loremIpsum } from '../Typography.mocks'
+import { Typography } from '..'
+
+const { BodyS, BodyM, BodyL } = Typography
+
+describe('BodyX Component', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+
+  test('renders BodyS correctly', () => {
+    const { asFragment } = render(<BodyS>{'Text'}</BodyS>)
+
+    const text = screen.getByText('Text')
```

Comment:
Can we add a role like 'paragraph' to be able to retrieve them in tests using roles?